### DISCUSSION TITLE
100pf passthru cap availability

### DISCUSSION THREAD

**Discussion by neilo40** [jasonsbeer/Amiga-2000-ATX#19] [General]
It looks like there's a global shortage of 1206 100pf passthru caps.  mouser, farnell, digikey are all backordered into next year.

There seems to be plentiful supply of 47pf and 220pf in 1206 size.  In a pinch, would either of these be ok?  would it be better to go up or down from 100pf, and what would the impact be?

**Comment by jasonsbeer** [comment]
In a pinch, go with the 47pF. An alternate part is NFE31PT101C1E9L. These are also out of stock at Digikey, but maybe you can track them down elsewhere. I saw them on Ali Express a couple weeks ago.
